CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# librdkafka NEXT
+
+## Fixes
+
+### General fixes
+
+ * Fix accesses to freed metadata cache mutexes on client termination (#3279)
```

Comment:
would this be causing segfault's on termination?
dotnet people have been complaining about that and i haven't investigated yet because it can happen if they are using the client after dispose, and people often do.

Reply:
Possible but unlikely, this was only caught in ClickHouse's TSAN tests.